migrator/zenodo_rdm_migrator/transform/records.py

```diff
@@ -89,3 +91,13 @@ def _transform(self, entry):
             "parent": self._parent(entry),
             "record_files": self._record_files(entry),
         }
+
+
+class ZenodoActionRecordTransform(ZenodoRecordTransform):
+    """Temporary hack until the json attribute loading is solved."""
```

Comment:
self note: for discussion, what's the issue this is solving?

Reply:
This was already there, so I'm not sure either 😅

Reply:
memory works: the problem is that debezium sends it as a string so we need to load it. probably a config to be changed.